Documentation/ceph-upgrade.md

```diff
@@ -387,6 +387,34 @@ At this point, your Rook operator should be running version `rook/ceph:v1.7.0`.
 
 Verify the Ceph cluster's health using the [health verification section](#health-verification).
 
+### **6. Update CephRBDMirror and CephBlockPool configs**
+
+If you are not using a `CephRBDMirror` in your Rook cluster, you may disregard this section.
+
+Otherwise, please note that the location of the `CephRBDMirror` `spec.peers` config has moved to
+`CephBlockPool` `spec.mirroring.peers` in Rook v1.7. This change allows each pool to have its own
+peer and enables pools to re-use an existing peer secret if it points to the same cluster peer.
+
+You may wish to see the [CephBlockPool spec Documentation](ceph-pool-crd.md#spec) for the latest
+configuration advice.
+
+The pre-existing config location in `CephRBDMirror` `spec.peers` will continue to be supported for
+this release. Users are required to migrate this setting from `CephRBDMirror` to
+relevant `CephBlockPool` resources but may do so any time before upgrading to Rook v1.8.
```

Comment:
@leseb I thought we won't be removing the `spec.peers` in `CephRBDMirror`.  Is that not the case now?  (asking because we are suggesting it would be removed in the upgrade doc)

Reply:
Yes, there is no plan to remove it concretely yet. So let's remove that statement for now :) .